core-blocks/shortcode/index.js

```diff
@@ -45,7 +44,7 @@ export const settings = {
 					text: {
 						type: 'string',
 						shortcode: ( attrs, { content } ) => {
-							return content;
+							return content.replace( /<br \/>/g, '' );
```

Comment:
How do we know if these are added by the browser? Guess we would need to check if it's plain text, and only remove then?

Btw, the current editor also destroys the following.

```html
[bwcsv]One,Two,Three
A,B,C
D,E,F
[/bwcsv]
```

becomes

```html
<p>[bwcsv]One,Two,Three A,B,C D,E,F [/bwcsv]</p>
```

Reply:
And if pasted in TinyMCE it becomes:

```html
<p>[bw_csv]One,Two,Three<br />A,B,C<br />D,E,F<br />[/bw_csv]</p>
```

Reply:
> Btw, the current editor also destroys the following.

Not if manually entered, right?

Regardless, a block whose purpose is to hold shortcodes should be capable of preserving arbitrary formatting, IMO.

Reply:
Yes, it should preserve it, agreed. But this line here is about input. Now I just realised this is actually the Markdown converter inserting these line break elements? With the `simpleLineBreaks` option. 😰